.github/workflows/delivery/docker/buildpack.yml

```diff
@@ -0,0 +1,16 @@
+go:
+  targets:
+    - ./cmd/pack
+
+  build:
+    # The go.build.flags property allows you to override the default build
+    # flags when compiling your program.
+    flags:
+      - -buildmode=default
+      - -tags=paketo
+      - -ldflags="-s -w -X 'github.com/buildpacks/pack.Version=#{PACK_VERSION}#' -extldflags ${LDFLAGS}"
```

Comment:
There is currently an open issue (https://github.com/paketo-buildpacks/go-build/issues/60) for interpolating as part of pack build, so we don't have to do it in the workflow.